services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/models.py

```diff
@@ -1,29 +1,59 @@
-from typing import Dict, List
+from typing import Optional
 
 from packaging import version
-from pydantic import BaseModel, ByteSize, Field
+from pydantic import BaseModel, ByteSize, Field, validator
 
 LEGACY_INTEGRATION_VERSION = version.Version("0")
 
 
 class ContainerHostConfig(BaseModel):
-    binds: List[str] = Field(
+    # NOTE: https://github.com/ITISFoundation/osparc-simcore/issues/3506
+    # Be careful! --priviledged, --pid=host --cap-add XXX should never be usable here!!
+    # at the moment they are not part of the possible configuration but if they were
+    # to, ensure they are properly validated
+
+    binds: list[str] = Field(
         ..., alias="Binds", description="A list of volume bindings for this container"
     )
     init: bool = Field(
-        True,
+        default=True,
```

Comment:
please remind me the difference between not having and having the default

Reply:
you do not have pyright/pylance in vscode? this is the reason why

Reply:
I see that pylance is installed No idea why this is not showing anything